server/people.js

```diff
@@ -19,6 +19,24 @@ const getPerson = field => async (client, id) => {
   return rows[0];
 };
 
+const addPerson = async (client, email, password) => {
+  const result = await client.query(
+    'INSERT INTO "Dootman"(id, email, password) VALUES ($1, $2, $3)',
+    [Date.now(), email, password]
+  );
+  console.log('insert result', result);
+  return result;
+};
+
+const deletePerson = async (client, email) => {
```

Comment:
We should be doing this by id, so change all instances of `email` to `id` in this function.